### PR TITLE
update for creating autopilot cluster

### PR DESCRIPTION
**Pull request by seanwilkens-ssc**
Steps to have the client create their own cluster and subnet.  Fix related to AZ bug 4768.

### CODE REVIEW

**Review by danielsvensson-ssc — DISMISSED** [review]

File: Developer/Guide.md [hunk 1]
```diff
@@ -81,6 +81,45 @@ SSC has built 4 distinct GCP organizations to isolate each environment from each
 
   ![experimentation networking](img/experimentation-networking.png)
 
+### AutoPilot Cluster Creation
+
+In the event a client wants to create an AutoPilot cluster for the experimentation landing-zone, the following configurations should be followed. *Note: that this is only for standard GKE clusters.*
```
Comment (lucstjean-ssc) [inline Developer/Guide.md:86]: I would rephrase this line *Note: that this is only for standard GKE clusters.* as it might be confusing. GKE clusters can be deployed in standard or autopilot mode. I meant to include the instructions for standard clusters as well as autopilot, and make a note that these instructions are not for config controller clusters deployed using `gcloud anthos config controller`.

Comment (seanwilkens-ssc) [reply Developer/Guide.md]: Excellent point.  Update to address comment has been submitted

**Review by lucstjean-ssc — APPROVED** [review]
/lgtm Nice!

**Review by danielsvensson-ssc — APPROVED** [review]
